packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx

```diff
@@ -0,0 +1,78 @@
+import { normalize } from '../../Styling';
+import { IChicletXsmallStyleProps, IChicletXsmallStyles } from './ChicletXsmall.types';
+
+export const getStyles = (props: IChicletXsmallStyleProps): IChicletXsmallStyles => {
+  const { theme, className } = props;
+  const { palette } = theme;
+
+  return {
+    root: [
+      'ms-ChicletXsmall',
+      theme.fonts.xSmall,
+      normalize,
+      {
+        WebkitFontSmoothing: 'antialiased',
+        backgroundColor: palette.white,
+        boxShadow: '0 1px 3px 0 rgba(0, 0, 0, 0.3)',
+        width: 250,
+        height: 103,
+        userSelect: 'none',
+        position: 'relative',
+        selectors: {
+          ':hover': {
+            cursor: 'pointer'
+          }
+        }
+      },
+      className
+    ],
+    icon: [
+      {
+        height: 30.32,
+        width: 28.64,
+        left: 3,
+        bottom: 36,
+        padding: '17.44px',
+        position: 'absolute',
+        backgroundColor: '	#DCDCDC'
```

Comment:
Recommend revising this as I don't think its valid unless we normalize and strip empty space. We should probably pull this color from the theme or similar? cc: @natalieethell